student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+# Student Class Inheriting From Person Class
+class Student < Person
+  def initialize(classroom)
+    super('student')
```

Comment:
- [ ] Your **Student** class inherits from **Person**, hence your **initialize** method here is missing important declarations like `age`, `name = "Unknown"`, `parent_permission = true`

- [ ] And your **super** method should pass up these 3 parameters to the person class, instead of 'student'